src/logics/radiant-v2/logic.borrow.ts

```diff
@@ -18,14 +18,14 @@ export class BorrowLogic extends core.Logic implements core.LogicTokenListInterf
 
   async getTokenList() {
     const service = new Service(this.chainId, this.provider);
-    const tokens = await service.getAssets();
+    const reserveTokens = await service.getBorrowTokens();
```

Comment:
suggest using the same name
 
` const borrowTokens = await service.getBorrowTokens();`

Reply:
updated, and move the aave-like the same logic to the modules